.gitignore

```diff
@@ -9,3 +9,7 @@ css/style.css.map
 env
 env*
 venv
+.vscode
+static/css/style.css
```

Comment:
Why are the css artifacts being git ignored?

Reply:
Hey @gsong , great question and call-out. 

You might know a lot more than me about this one! Here's where I was coming from:

Just to ensure we're on the same page, there's the compiled `style.css` files we shouldn't touch as we write code, and then there's the `style.scss` where we should be making all of our changes.

For the same reason we usually don't include things like node modules, a `dist` folder, or other build artifacts, I've seen it be a best practice to not commit the compiled `.css` files. 

This StackOverflow article discusses a few other reasons it's usually best to avoid committing them:
https://stackoverflow.com/questions/26933706/is-it-considered-harmful-to-commit-readable-css-in-addition-to-scss-source/26933931

You are right that they are necessary for a deployed app to leverage, but these are typically re-generated in the deployment process every time from the base .scss files. As a result, committing these shouldn't be necessary for the deployment.

That said - I'm not super familiar with the deploy process being used so can't say with 100% certainty, maybe you can!

The "good" news is that, frankly, adding them to the .gitignore now does nothing on its own. My understanding is that [once Git is tracking a file, you need to be more forceful to stop tracking it](https://www.git-tower.com/learn/git/faq/ignore-tracked-files-in-git).

Happy to revert this part if there were some things we didn't consider.

Reply:
You're right in the *typical* deployment process. But Techtonica doesn't have any CI/CD in place so everything is manual. See https://github.com/Techtonica/techtonica.org#deployment-to-dreamhost for deployment details.

My suggestion would be to do one of the following (in order of preference):

1. Revert the change—as it stands this introduces confusion.
2. Update the deployment instructions to include an additional Sass build step.
3. Go full in and add a GitHub Action to deploy the `master` branch.

I advise against 2 & 3 because as I understand it, the website is going to change, so the infrastructure may change anyway.